package.json

```diff
@@ -12,9 +12,16 @@
   "bugs": {
     "url": "https://github.com/adazzle/react-data-grid/issues"
   },
-  "main": "lib/index.js",
-  "module": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "exports": {
+    "node": {
+      "import": "./lib/bundle.mjs",
+      "require": "./lib/bundle.cjs"
+    }
+  },
+  "browser": "./lib/index.js",
+  "main": "./lib/bundle.cjs",
+  "module": "./lib/index.js",
```

Comment:
`"main"` will work for Node 10 (maintenance).
`"browser"` and `"module"` are ignored by Node.
https://nodejs.org/docs/latest-v10.x/api/esm.html
https://github.com/nodejs/Release#release-schedule

webpack and rollup should pick up `"browser"` and `"module"` before `"main"` by default.
https://webpack.js.org/configuration/resolve/#resolvemainfields
https://github.com/rollup/plugins/tree/master/packages/node-resolve#mainfields